### PR TITLE
Update branding to 3.1.8

### PR DESCRIPTION
**Pull request by dougbu**

### CODE REVIEW

File: eng/Versions.props [hunk 1]
```diff
@@ -52,11 +52,11 @@
     <SystemDiagnosticsDiagnosticSourcePackageVersion>4.7.1</SystemDiagnosticsDiagnosticSourcePackageVersion>
   </PropertyGroup>
   <PropertyGroup Label="Dependencies from dotnet/core-setup">
-    <MicrosoftDotNetPlatformAbstractionsPackageVersion>3.1.6</MicrosoftDotNetPlatformAbstractionsPackageVersion>
-    <MicrosoftExtensionsDependencyModelPackageVersion>3.1.6</MicrosoftExtensionsDependencyModelPackageVersion>
+    <MicrosoftDotNetPlatformAbstractionsPackageVersion>3.1.3</MicrosoftDotNetPlatformAbstractionsPackageVersion>
+    <MicrosoftExtensionsDependencyModelPackageVersion>3.1.3</MicrosoftExtensionsDependencyModelPackageVersion>
```
Comment (dougbu) [inline eng/Versions.props:56]: Something is going haywire here. 3.1.6 versions of these packages exist on NuGet.org. So, why was 3.1.7 built against 3.1.3 packages❔ 

/cc @wtgodbe @mmitche @rbhanda  @silvioguiso

Comment (dougbu) [reply eng/Versions.props]: Figured out offline. Problem relates to coherent parent dependencies and lack of strict coherency in Arcade's 3.x releases.

**Comment by dougbu** [comment]
FYI builds are failing known reasons and @wtgodbe is working on the resolution

**Comment by dougbu** [comment]
@ajcvickers @smitpatel @bricelam what's going on w/ the Sqlite functional tests on macOS❔ The same test assembly failed twice so far without any reported test failures. And, for example, https://dev.azure.com/dnceng/_apis/resources/Containers/4808845/Windows_NT_Windows?itemPath=Windows_NT_Windows%2FEFCore.Sqlite.FunctionalTests_netcoreapp3.1_x64.log looks clean.

**Comment by smitpatel** [comment]
Most likely https://github.com/dotnet/efcore/pull/21761 needs to be applied to release/3.1 too. From what I remember, it causes seg fault so nothing in logs.

**Comment by dougbu** [comment]
🆗 that is a test-only fix so I'll cherry-pick it here. Thanx @smitpatel

**Pull request merged**